pxtblocks/blocklyloader.ts

```diff
@@ -11,7 +11,9 @@ namespace pxt.blocks {
         images: 45,
         variables: 330,
         text: 160,
-        lists: 260
+        lists: 260,
+	math: 230,
```

Comment:
If you install the (vim)[https://marketplace.visualstudio.com/items?itemName=vscodevim.vim] extension, you should not be too lost. You'll get the best TypeScript experience in VSCode.